src/main/java/io/github/mapepire_ibmi/Pool.java

```diff
@@ -59,21 +50,8 @@ public Pool(PoolOptions options) {
      * size.
      *
      * @return A CompletableFuture that resolves when all jobs have been created.
-     * @throws UnknownServerException
-     * @throws SQLException
-     * @throws URISyntaxException
-     * @throws ExecutionException
-     * @throws InterruptedException
-     * @throws NoSuchAlgorithmException
-     * @throws JsonProcessingException
-     * @throws KeyManagementException
-     * @throws JsonMappingException
-     * @throws ClientException
      */
-    public CompletableFuture<Void> init()
-            throws JsonMappingException, KeyManagementException, JsonProcessingException, NoSuchAlgorithmException,
-            InterruptedException, ExecutionException, URISyntaxException, SQLException, UnknownServerException,
-            ClientException {
+    public CompletableFuture<Void> init() throws Exception {
```

Comment:
Why change to throw generic exception instead of specific exceptions?

Reply:
For simplicity, I opted for simply throwing the generic `Exception` as the application using these APIs will likely not have separate handlers for each of these different exceptions. If they want this granularity still, they can still catch with the specific exception.

Reply:
You can also see here in the documentation, we have outlined the most important exceptions that they will be handling under the `Exception Handling` section: https://mapepire-ibmi.github.io/guides/usage/java/